### ISSUE TITLE
[Feature Request] Store the nsec as ncryptsec

### ISSUE THREAD

**Issue by antonioconselheiro** [fiatjaf/nos2x#55]
I'm requesting to include the encrypted nsec (ncryptsec) and request the password in the client to open the extension. I'm requesting this because it bothers me that my nsec is available for copying and viewing just by pressing the button "show key".

I understand that it would be less problematic to temporarily store a password capable of opening the encrypted content than to directly store the nsec permanently.

This make sense?

**Comment by fiatjaf** [comment]
Yes, will implement eventually.

**Comment by fiatjaf** [comment]
@manimejia I think these two requirements are implemented, no?

However the secret key is stored in plaintext, not encrypted. I am not sure how to properly implement it so it is stored encrypted, but maybe I'll do that eventually.

**Comment by fiatjaf** [comment]
No, you can paste it encrypted. I'm pretty sure. I can't test now because I can't delete my key from the extension, but I know I wrote this code in the latest release. Maybe it is broken. I'll check later.

**Comment by ssbright** [comment]
Hello, I wanted to follow up with this question. How do I sign into a nostr app, like Primal, with a ncryptsec? Do I have to decrypt? If so, how. If not, how do I enter my password to decrypt the ncryptsec?

**Comment by fiatjaf** [comment]
@mbrat21

**Comment by fiatjaf** [comment]
Did you try installing nos2x and inputting the ncryptsec on the options page?

**Comment by ssbright** [comment]
this works! thank you. Any suggestions on how I could do this on mobile so I could login safely on a phone?
